javascript-forms/form-values.js

```diff
@@ -0,0 +1,11 @@
+var $form = document.querySelector('#contact-form');
+$form.addEventListener('submit', function (event) {
+  event.preventDefault();
+  var valueObj = {
+    name: $form.elements.name.value,
+    email: $form.elements.email.value,
+    message: $form.elements.message.value
+  };
+  console.log('valueObj:', valueObj);
+  document.querySelector('#contact-form').reset();
```

Comment:
You already have a variable holding the form element, you don't need to query for it again.